ui/src/app/applications/components/application-resource-tree/application-resource-tree.tsx

```diff
@@ -513,8 +768,17 @@ export const ApplicationResourceTree = (props: ApplicationResourceTreeProps) =>
             findNetworkTargets(networkNodes, parent.networkingInfo).forEach(child => {
                 const children = childrenByParentKey.get(treeNodeKey(parent)) || [];
                 hasParents.add(treeNodeKey(child));
-                children.push(child);
-                childrenByParentKey.set(treeNodeKey(parent), children);
+                if (child.kind !== 'Pod') {
```

Comment:
I think we can avoid repeating these lines:
```
children.push(child);
childrenByParentKey.set(treeNodeKey(parent), children);
```

By doing something like:
```
if (!props.showCompactNodes || child.kind !== 'Pod') {
    // those lines
} else {
    processPodGroup(parent, child, props);
}

Reply:
Thanks Remington.  Agreed.